### PR TITLE
add new snippets, bump to 0.0.3

### PR DESCRIPTION
**Pull request by obweger**

### CODE REVIEW

File: snippets/snippets.md [hunk 1]
```diff
@@ -42,6 +44,28 @@ export const createFoo = (): CreateFooAction => ({
 });
 ```
 
+## View
```
Comment (marcins) [inline snippets/snippets.md]: Are you adding this docco manually? Would be nice if we could auto-generate it from the snippets themselves :)

File: snippets/javascript/imports-view.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		"description": "Imports React",
+		"body": [
+			"import React from 'react';$0"
+		]
+	},
+	"React: Component": {
+		"prefix": "import-react-component",
+		"description": "Imports 'Component' from React",
+		"body": [
+			"import React, { Component } from 'react';$0"
```
Comment (marcins) [inline snippets/javascript/imports-view.json]: I assume this isn't additive, and you need to choose one of the `import`s when you start your file? Is there a lot of value to having all these different variants? (maybe there is!)

File: snippets/snippets.md (hunk 1 above)
Comment (obweger) [reply snippets/snippets.md]: yeah would - maybe next tiem :)

File: snippets/javascript/imports-view.json (hunk 2 above)
Comment (obweger) [reply snippets/javascript/imports-view.json]: so there are eslint rules in place that just automerge different imports together. so you can really just import when you need something, press save, and :done:.

**Pull request merged**